### PR TITLE
No more dallas spam & dallas and purger delay fix

### PR DESCRIPTION
**Pull request by ValoTheValo**
## About The Pull Request
this PR makes it so that the Dallas no longer spawns in the guild radio when upgraded. also removes its wield delay as its not supposed to have it (it was removed from all LMGs a long while ago)

## Why It's Good For The Game
the rarest and strongest gun in the game shouldn't be common as dirt

## Testing
i went in game, upgraded the radio with the siphon and waited. no dallases spawned

## Changelog
:cl:
balance: Dallas no longer spawns in guild radio when upgraded
fix: Dallas no longer has a delay on wielding, dallas now gives gun parts correctly 
balance: Purger gun no longar has a delay on wielding
/:cl:

### CODE REVIEW

**Comment by DiaFRAME444** [comment]
That list seems a bit empty now, no additions?

**Comment by maikilangiolo** [comment]
Guild players on suicide watch.



> That list seems a bit empty now, no additions?

The radio shouldn't spawn shit, just another inane spcr change.

**Comment by thevandie** [comment]
Based pr
I was kind of abusing it today but I agree it shouldn't exist

**Comment by thevandie** [comment]
Purger and dominion also still have the delay, fix those too pls

**Comment by ValoTheValo** [comment]
> Purger and dominion also still have the delay, fix those too pls

adressed

**Comment by ValoTheValo** [comment]
> Purger and dominion also still have the delay, fix those too pls

also only purger had delay, dominion didnt. then again those weapons are super easy to get mixed up

**Comment by Firefox13** [comment]
Probably should remove stranger's plasma gun since it's also OP.

**Comment by ValoTheValo** [comment]
> Probably should remove stranger's plasma gun since it's also OP.

done

File: code/modules/projectiles/guns/energy/plasma.dm [hunk 1]
```diff
@@ -56,8 +56,6 @@
 	slot_flags = SLOT_BACK
 	fire_delay = 20
 	charge_cost = 200
-	wield_delay = 0.8 SECOND
-	wield_delay_factor = 0.9 // 90 vig for instant wield
```
Comment (hyperioo) [inline code/modules/projectiles/guns/energy/plasma.dm:60]: Nice try to sneak an unmentionned change 👀

**Review by hyperioo — CHANGES_REQUESTED** [review]

**Review by hyperioo — APPROVED** [review]

**Pull request merged**